docs/modules/stackablectl/pages/commands/completions.adoc

```diff
@@ -0,0 +1,86 @@
+= stackablectl completions
+
```

Comment:
Add something like "This command is only used to generate shell completions and normally doesn't need to be used by the user"

Reply:
Will be added in a follow-up PR.